internal/daysteps/daysteps.go

```diff
@@ -12,9 +17,46 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	parts := strings.Split(data, ",")
+	if len(parts) != 2 {
+		return 0, 0, &strconv.NumError{Func: "parsePackage", Num: data, Err: strconv.ErrSyntax}
+	}
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return 0, 0, err
```

Comment:
Ошибки лучше оборачивать
```suggestion
		return 0, 0, fmt.Errorf("invalid steps format: %w", err)
```